app/src/test/java/edu/uc/kansakpb/ikeahjalpare3/FurnitureDataUnitTest.kt

```diff
@@ -63,19 +78,18 @@ class FurnitureDataUnitTest {
     }
 
     private fun thenResultContainsDrawerDresser() {
-        var KopangDrawerDresserFound = false;
-        mvm.furnitures.observeForever {
+        var kopangDrawerDresserFound = false;
```

Comment:
Local variable names should start with a lowercase character.

Reply:
Yes.